setup.cfg

```diff
@@ -1,3 +1,3 @@
 [tool:pytest]
-addopts = --tb=short --ds=carbondoomsday.settings
-testpaths = carbondoomsday/carbondioxide/tests
+addopts = --tb=short --ds=carbondoomsday.settings --disable-pytest-warnings
+testpaths = ./carbondoomsday/carbondioxide/test/
```

Comment:
Remove "./" here, too?!
Also `--disable-pytest-warnings`.
(Probably commented on the commit before)